ci-operator/config/openshift/verification-tests/openshift-verification-tests-master__installation-nightly-4.14.yaml

```diff
@@ -70,6 +70,37 @@ tests:
     test:
     - ref: cucushift-installer-check-cluster-health
     workflow: cucushift-installer-rehearse-aws-ipi-preserve-bootstrap-ign
+- as: azure-ipi-disktype-premium-lrs-f28
+  cron: 17 0 24 * *
+  steps:
+    cluster_profile: azure-qe
+    env:
+      AZURE_DEFAULT_MACHINE_DISK_TYPE: Premium_LRS
+      BASE_DOMAIN: qe.azure.devcluster.openshift.com
+    test:
+    - chain: cucushift-installer-check-cluster-health
+    workflow: cucushift-installer-rehearse-azure-ipi-disktype
+- as: azure-ipi-disktype-standardssd-lrs-f28
+  cron: 54 6 12 * *
+  steps:
+    cluster_profile: azure-qe
+    env:
+      AZURE_COMPUTE_DISK_TYPE: StandardSSD_LRS
+      AZURE_CONTROL_PLANE_DISK_TYPE: StandardSSD_LRS
+      BASE_DOMAIN: qe.azure.devcluster.openshift.com
+    test:
+    - chain: cucushift-installer-check-cluster-health
+    workflow: cucushift-installer-rehearse-azure-ipi-disktype
+- as: azure-ipi-disktype-worker-standard-lrs-f28
+  cron: 55 9 4 * *
+  steps:
+    cluster_profile: azure-qe
+    env:
+      AZURE_COMPUTE_DISK_TYPE: Standard_LRS
+      BASE_DOMAIN: qe.azure.devcluster.openshift.com
```

Comment:
so `Standard_LRS` only could be used for compute node, but not masters?

Reply:
yes, it only supports StandardSSD_LRS and Premium_LRS on master